ext/standard/array.c

```diff
@@ -3869,15 +3875,15 @@ static zend_always_inline void php_array_merge_wrapper(INTERNAL_FUNCTION_PARAMET
 }
 /* }}} */
 
-/* {{{ proto array array_merge(array arr1 [, array ...])
+/* {{{ proto array array_merge([, array ...])
```

Comment:
I don't think there should be a leading comma here.

Reply:
Adjusted.